CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
+ - Added notification under reward panel that appears three days before contribution if wallet has insufficient BAT. ([#1479](https://github.com/brave/brave-browser/issues/1479))
+ - Added recurring date within the tip table under Brave Rewards. ([#1834](https://github.com/brave/brave-browser/issues/1834))
+ - Cleaned up default search engines settings. ([#3316](https://github.com/brave/brave-browser/issues/3316))
+ - Changed rewards publisher list being fetched every 48 hours to 24 hours. ([#3171](https://github.com/brave/brave-browser/issues/3171))
+ - Updated new tab page background images and refined design. ([#2504](https://github.com/brave/brave-browser/issues/2504))
+ - Updated icons under new tab page. ([#1593](https://github.com/brave/brave-browser/issues/1593))
+ - Fixed summary under rewards panel being cleared when restarting Brave. ([#3055](https://github.com/brave/brave-browser/issues/3055))
+ - Fixed insufficient funds notification being displayed when wallet has more than enough BAT. ([#3212](https://github.com/brave/brave-browser/issues/3212))
+ - Fixed being able to tip even though Brave Rewards is disabled. ([#2214](https://github.com/brave/brave-browser/issues/2214))
+ - Fixed auto-contribution table not working if "Minimum page time before logging visit" is set to one minute. ([#3320](https://github.com/brave/brave-browser/issues/3320))
+ - Fixed HTTPS Everywhere not working correctly on some websites. ([#1535](https://github.com/brave/brave-browser/issues/1535))
+ - Fixed public content settings not being inherited by private window. ([#1373](https://github.com/brave/brave-browser/issues/1373))
+ - Fixed sizing of several action icons that appear in the URL bar. ([#2295](https://github.com/brave/brave-browser/issues/2295))
+ - Fixed tipping option in certain cases not being displayed under rewards panel. ([#2089](https://github.com/brave/brave-browser/issues/2089))
+ - Fixed Twitch icons under rewards panel not being displayed in certain cases. ([#1521](https://github.com/brave/brave-browser/issues/1521))
+ - Fixed whitespace at the beginning or end of the recovery key prevents successful wallet recovery. ([#2665](https://github.com/brave/brave-browser/issues/2665))
+ - Fixed being able to tip YouTube publishers in certain cases even though "Allow contributions to videos" is disabled. ([#2463](https://github.com/brave/brave-browser/issues/2463))
+ - Fixed URL and icons under reward panel mismatching in some cases. ([#2073](https://github.com/brave/brave-browser/issues/2073))
+ - Fixed "add funds" link not working under rewards panel when wallet has insufficient BAT for tip. ([#2628](https://github.com/brave/brave-browser/issues/2628))
+ - Fixed not being able to authenticate (2FA with Google prompt) with Gmail. ([#1356](https://github.com/brave/brave-browser/issues/1356))
```

Comment:
-> Fixed 2FA with Google Accounts when Brave Shields are enabled.  (could leave as is since majority of people will sign-in to Gmail, was also thinking "Fixed 2FA with Google products when...")